src/components/Footer/Footer.js

```diff
@@ -1,6 +1,170 @@
 import React from 'react';
+import * as S from './FooterStyle';
 
 function Footer() {
-  return <h1>Footer 입니다.</h1>;
+  return (
+    <S.Footer>
+      <S.FooterTop>
+        <S.FooterTopWrap>
+          {FooterTopData.map(data => {
+            return (
+              <S.FooterTopList key={data.id}>
+                <S.FooterTopListText>{data.text}</S.FooterTopListText>
+              </S.FooterTopList>
+            );
+          })}
```

Comment:
```suggestion
          {FooterTopData.map(({id, text}) => {
            return (
              <S.FooterTopList key={id}>
                <S.FooterTopListText>{text}</S.FooterTopListText>
              </S.FooterTopList>
            );
          })}
```
이렇게 구조분해할당 적용할 수도 있습니다!